web/build.gradle

```diff
@@ -1,27 +0,0 @@
-def eclipsePluginProjects = subprojects.findAll{it.file('build.properties').exists()}
-def exampleProjects = subprojects.findAll{it.name.contains("example")}
-
-configure(eclipsePluginProjects) {
-	apply from: "${rootDir}/gradle/eclipse-plugin-project.gradle"
-}
-
-configure(subprojects - eclipsePluginProjects) {
-	apply plugin: 'java'
-	apply from: "$rootDir/gradle/eclipse-settings.gradle"
-
-	sourceSets {
-		main.java.srcDir 'src/main/xtext-gen'
-		main.java.srcDir 'src/main/xtend-gen'
-		test.java.srcDir 'src/test/xtend-gen'
-	}
-	
-	if (JavaVersion.current().isJava8Compatible()) {
```

Comment:
I guess you are referring to the JavaDoc line? That seems to have been a workaround for a broken JavaDoc classpath. I don't see any problems without it.